ui/src/notebooks/pipes/Query/ResultsResizer.tsx

```diff
@@ -0,0 +1,133 @@
+// Libraries
+import React, {FC, useRef, useEffect, ReactNode, useState} from 'react'
+import {round} from 'lodash'
```

Comment:
i'm drawing a line right here....
use host functions, they're faster, free on resources, and you're not going to need to use hundredths of a pixel, get lodash out of here
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/round

Reply:
Why is there no "grimace" emoji in github??